DTXMania/Code/Stage/07.Performance/GuitarScreen/CActPerfGuitarBonus.cs

```diff
@@ -64,7 +64,7 @@ public override unsafe int OnUpdateAndDraw()
 
                 if (CDTXMania.ConfigIni.bShowScore)
                 {
-                    drawBonusScoreAnimation(EInstrumentPart.GUITAR, new Point(303, 45));
+                    drawBonusScoreAnimation(EInstrumentPart.GUITAR, new Point(333, 45));
```

Comment:
omg never ending hard coded values

Reply:
Drawing positions of textures are all hard-coded in this project. Will need to major overhaul and refactoring if we want to make it configurable or auto-scale with screen resolution.